databases/week2/data3.sql

```diff
@@ -0,0 +1,29 @@
+--More queries
+
+USE hyf_lesson1;
+
+--%spotify
+SELECT *
+FROM user
+LEFT JOIN task ON user.id = task.user_id
+WHERE user.email LIKE '%spotify.com';
+
+--Donald_duck
+SELECT user.name AS user_name, task.title AS the_task, task.status_id, status.name AS current_status
+FROM user
+JOIN task          -- when I put condition ON user.id = task.user_id it doesn't give me rows in the table
+JOIN status ON status.id = task.status_id
+WHERE user.name LIKE 'donald duck' 
+AND status_id = 1;
+
+--Maryrose Meadows
+SELECT user.name, task.title, task.created
+FROM user
+LEFT JOIN task ON user.id = task.user_id
+WHERE user.name LIKE 'Maryrose Meadows'
+AND month(created)= 9;
```

Comment:
remember when you use like you to add % inside of quotes. ex: instead of ` Like 'Maryrose Meadows'` write ` Like '%Maryrose Meadows%'`. And also double check your query if it works.

Reply:
Oh sure, sorry I forgot that. :)